src/queries/conditions.tsx

```diff
@@ -1,8 +1,30 @@
 import gql from 'graphql-tag'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const DEFAULT_ORDER_FIELD = 'resolveBlockNumber'
+
+export const ConditionsSearchQuery = gql`
+  query Conditions($conditionId: String!) {
+    conditions(
+      first: 1000
+      where: { id: $conditionId }
+      orderBy: $DEFAULT_ORDER_FIELD
+      orderDirection: desc
+    ) {
+      id
+      oracle
+      questionId
+      outcomeSlotCount
+      resolved
+      creator
+      resolveBlockNumber
+    }
+  }
+`
+
 export const ConditionsListQuery = gql`
   query Conditions {
-    conditions(first: 1000) {
+    conditions(first: 1000, orderBy: $DEFAULT_ORDER_FIELD, orderDirection: desc) {
```

Comment:
The variable `DEFAULT_ORDER_FIELD` I think is not needed, and is not passed as a variables in the useQuery.
I dont know what exactly this variable does.
We can see in the image the variable is not used, any thoughts about it @hpmaxi @lmcorbalan ?
![Screenshot_20200820_170544](https://user-images.githubusercontent.com/1144028/90820199-67902e80-e307-11ea-896f-8fb19fa62074.png)